packages/neon/neon/lib/src/widgets/cached_image.dart

```diff
@@ -160,8 +169,21 @@ class _NeonCachedImageState extends State<NeonCachedImage> {
               width: widget.size?.width,
               fit: widget.fit,
               gaplessPlayback: true,
+              errorBuilder: (final context, final error, final stacktrace) => _buildError(error),
             );
           },
         ),
       );
+
+  Widget _buildError(final dynamic error) =>
+      widget.errorBuilder?.call(context, error) ??
+      NeonException(
+        error,
+        onRetry: () {
+          setState(() {});
+        },
+        onlyIcon: true,
+        iconSize: widget.size?.shortestSide,
+        color: widget.iconColor ?? Theme.of(context).colorScheme.error,
```

Comment:
I think the error should always be displayed in the error color. Otherwise it might not get noticed.